server/index.js

```diff
@@ -3,7 +3,18 @@ const typeDefs = require('./schema');
 const resolvers = require('./resolvers');
 const catAPI = require('./datasources/cat-api');
 
+const normalizeCamelToSnakeCase = (string) => (
+  words(toString(string).replace(/['\u2019]/g, '')).reduce((result, word, index) => (
+    result + (index ? '_' : '') + word.toLowerCase()
+  ), '')
+)
```

Comment:
Oh, wow. :D

What happens if we have `something__like___that` or `__that`? A couple of unit tests covering different scenarios would be great.